app/controllers/api/account.php

```diff
@@ -1076,39 +1101,40 @@
 
         $queueForEvents->setPayload(
             $response->output(
-                $token->setAttribute('secret', $loginSecret),
+                $token->setAttribute('secret', $tokenSecret),
                 Response::MODEL_TOKEN
             )
         );
 
         // Hide secret for clients
-        $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? $loginSecret : '');
+        $token->setAttribute('secret', ($isPrivilegedUser || $isAppUser) ? $tokenSecret : '');
 
         $response
             ->setStatusCode(Response::STATUS_CODE_CREATED)
             ->dynamic($token, Response::MODEL_TOKEN)
         ;
     });
 
-App::put('/v1/account/sessions/magic-url')
-    ->desc('Create magic URL session (confirmation)')
-    ->groups(['api', 'account', 'session'])
+App::put('/v1/account/sessions/token')
+    ->alias('/v1/account/sessions/magic-url')
+    ->alias('/v1/account/sessions/phone')
+    ->desc('Update token session')
+    ->groups(['api', 'account'])
     ->label('scope', 'public')
-    ->label('event', 'users.[userId].sessions.[sessionId].create')
-    ->label('audits.event', 'session.update')
+    ->label('auth.type', 'token')
```

Comment:
This token needs to be added in the following places
1. https://github.com/appwrite/appwrite/blob/342deb49896842ace0a80b60da1d5048a8dd993e/app/controllers/shared/api.php#L280
2. https://github.com/appwrite/appwrite/blob/342deb49896842ace0a80b60da1d5048a8dd993e/app/controllers/shared/api/auth.php#L27

In addition I also see that the `phone` auth type is missing in the above two places. You might wanna add it there too 🤔